app/src/lib/git/core.ts

```diff
@@ -182,9 +182,14 @@ function getDescriptionForError(error: DugiteError): string {
     case DugiteError.SSHAuthenticationFailed:
     case DugiteError.SSHPermissionDenied:
     case DugiteError.HTTPSAuthenticationFailed:
-      return `Authentication failed. You may not have permission to access the repository or the repository may have been archived. Open ${
-        __DARWIN__ ? 'preferences' : 'options'
-      } and verify that you're signed in with an account that has permission to access this repository.`
+      return `Authentication failed. Some common reasons include:
+- You are not logged in to your account: see ${
```

Comment:
Looking at that last screenshot, maybe a blank like before this first bullet point might help break it up a bit? Not a strong opinion.